prow/jobs/jobs.yaml

```diff
@@ -6,4 +6,26 @@ periodics:
     serviceAccountName: pre-submit-service-account
     containers:
     - image: alpine
-      command: ["/bin/date"]
\ No newline at end of file
+      command: ["/bin/date"]
+
+postsubmits:
+  #TODO(vijat@): Add support for all service controllers
+  #TODO(vijat@): Find a programmatic way to generate postsubmit jobs for all the service controllers
+  aws-controllers-k8s/apigatewayv2-controller:
+  - name: apigatewayv2-post-submit-release
+    decorate: true
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: true
+    spec:
+      serviceAccountName: post-submit-service-account
+      containers:
+        - image: errorbyte/ack-prow-cd #TODO(vijat@): find the correct place for hosting this image.
+          securityContext:
+            privileged: true
+          command: ["/bin/bash"]
+          args: ["-c", "cd cd/scripts && ./release-controller.sh"]
```

Comment:
Should this be the default entrypoint? I worry that if we make a change to the image we will have to remember to come back into here and modify all of the commands